styles/headsup/layout.s2

```diff
@@ -225,12 +227,14 @@ var string header_background_height;
     }
 
 var string headerimage_area_height;
```

Comment:
I think it's the same indentation as the other variables set in function print_stylesheet () so no.